meta/src/optimizer/mod.rs

```diff
@@ -105,6 +104,7 @@ pub enum OptimizedExpr {
     Choice(Box<OptimizedExpr>, Box<OptimizedExpr>),
     Opt(Box<OptimizedExpr>),
     Rep(Box<OptimizedExpr>),
+    RepOnce(Box<OptimizedExpr>),
```

Comment:
`enum OptimizedExpr` wasn't annotated with `#[non_exhaustive]`, so adding a new variant to it is unfortunately a semver breaking change